codemod/spotlight_util.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2007-2008 Facebook
+
+from Cocoa import *
```

Comment:
If this requires another module to be installed, I think you'll need to add it to `setup.py`. I think you might be able to conditionally require another module, again using `if platform.system() == 'Darwin'`.